### PR TITLE
Ilona coveralls

### PR DESCRIPTION
**Pull request by ilonamm**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test:
```
Comment (batmany13) [inline circle.yml]: we're using "circleci 2.0", so there's no longer a circle.yml file.  Take a look at .circleci/config.yml for the right place to put this stuff

File: .coveralls.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+repo_token: $coveralls_repo_token
```
Comment (batmany13) [inline .coveralls.yml:1]: how does this variable get set?

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+  override:
+    - mix coveralls.circle
```
Comment (batmany13) [inline circle.yml]: does this work?  I tried running this locally, and it blew up

**Review by batmany13 — CHANGES_REQUESTED** [review]
just need to change it to support circleci 2.0, and remove the circle.yml

File: .coveralls.yml (hunk 2 above)
Comment (ilonamm) [reply .coveralls.yml:1]: set it up in circleci as env variable

**Comment by ilonamm** [comment]
This should work now, except for some dogma errors in code.

**Comment by ilonamm** [comment]
should be all fixed, ready for @batmany13!

**Review by batmany13 — APPROVED** [review]

**Comment by ilonamm** [comment]
[Set up github + circleCI + Heroku](https://trello.com/c/Dcd4hT9i/3-set-up-github-circleci-heroku)

**Pull request merged**